### DISCUSSION TITLE
Change default editor font to Inter

### DISCUSSION THREAD

**Discussion by ettiSurreal** [godotengine/godot-proposals#9012] [Editor]
Related: https://github.com/godotengine/godot-proposals/issues/3764

Currently the editor uses Noto Sans as the default font. 
I'm here to propose to move to [Inter](https://rsms.me/inter/), a modern, open source font designed for interfaces.
I may be biased, but I think it looks nicer and more readable than Noto.

| Display Size | Noto Sans (Current) | Inter (Proposed) |
|--------|--------|--------|
| 100% | ![InterProposal1](https://github.com/godotengine/godot-proposals/assets/98223724/fd635b75-38ae-4805-9adb-28194b867f38) | ![InterProposal4](https://github.com/godotengine/godot-proposals/assets/98223724/c38a6f8d-84f4-45ee-b101-fcc223d17dcd) |
| 75% | ![InterProposal2](https://github.com/godotengine/godot-proposals/assets/98223724/898cba5b-eb34-48a1-a9c2-eac6e4284627) |![InterProposal5](https://github.com/godotengine/godot-proposals/assets/98223724/7d800a09-57ef-480f-ab0c-e316b693f100)|
| 50% | ![InterProposal3](https://github.com/godotengine/godot-proposals/assets/98223724/126acb89-a281-4678-a44e-a7da1b273d95) |![InterProposal6](https://github.com/godotengine/godot-proposals/assets/98223724/dc913a2e-9956-43ee-b457-0ed5d0ce936d)|

This is inspired by Soundcloud and Blender relatively recently using to Inter, so i used the OpenType features that mirror the latter.
cv02 - Open four
cv03 - Open six
cv04 - Open nine
cv05 - Lowercase L with tail
cv08 - Uppercase I with serif
cv09 - flat top three
tnum - Evenly spaced numbers, makes the interface less "jumpy".
![Godot_v4 2 1-stable_win64_dAKk3ZtmB3](https://github.com/godotengine/godot-proposals/assets/98223724/8d345635-0000-4047-b9f7-e9d8a09e9e6c)
![Godot_v4 2 1-stable_win64_TgusAfgfNy](https://github.com/godotengine/godot-proposals/assets/98223724/fbcf0d60-1db3-4fce-9d51-dc8edc5f0a25)

**Comment by Calinou** [comment]
I tend to agree that Inter is more readable than Noto Sans, but remember that it also visually looks a bit larger, which helps it be more readable. Considering there's a regular demand for smaller editor fonts by default, I'm not sure if going in that direction would be a good thing unless we also reduce the default font size. Can you take screenshots of the editor with the font size set to 12 and 13?

Also, one thing Noto has over Inter is visual consistency across languages. Noto has a *huge* selection of fonts to choose from (one of the largest in open source fonts). Inter looks nicer, but the second you have CJK or Arabic characters on screen, they'll look different than the rest of the editor.

> tnum - Evenly spaced numbers, makes the interface less "jumpy".

Note that regarding this, we could choose to use the code font (which is fixed-width) for numeric fields in the inspector.

**Reply by Zireael07** [reply]
+1 for using fixed with font for numeric fields

**Reply by ettiSurreal** [reply]
![image](https://github.com/godotengine/godot-proposals/assets/98223724/75f5a41f-9325-410f-bb44-61580cffa7a0)
![image](https://github.com/godotengine/godot-proposals/assets/98223724/8186cfbb-1f24-4fff-b8f6-3a7e6dcf7c9c)

Discovery while testing Inter: Not everything updates properly when you change theme parameters, specifically the size of tabs can break, and the font of categories doesn't update until you reopen the window.

**Reply by Calinou** [reply]
I think it looks nice at size 13, while also having more screen real estate available than with Noto Sans at size 14. This would need further testing on small displays like laptops to make sure it's still usable though.

**Comment by Repiteo** [comment]
Implemented in godotengine/godot#111140

**Comment by DeeJayLSP** [comment]
> Implemented in https://github.com/godotengine/godot/pull/111140.

Note: a few suggestions like alternate digits (from `cv02` to `cv04` + `cv09`) and default font size to 13 were left out.
